day_two_guide/topics/proc_backing-up-etcd.adoc

```diff
@@ -247,3 +247,35 @@ any external storage location.
 In the case of an all-in-one cluster, the etcd data directory is located in
 the `/var/lib/origin/openshift.local.etcd` directory.
 --
+* If etcd runs as a static pod, run the following commands:
++
+[IMPORTANT]
+====
+If you use static pods, use the v3 API.
+====
+.. Obtain the etcd endpoint IP address from the static pod manifest:
++
+----
+$ export ETCD_POD_MANIFEST="/etc/origin/node/pods/etcd.yaml"
+$ export ETCD_EP=$(grep https ${ETCD_POD_MANIFEST} | cut -d '/' -f3)
+----
+
+.. Obtain the etcd pod name:
++
+----
+$ oc login -u system:admin
+$ export ETCD_POD=$(oc get pods -n kube-system | grep -o -m 1 '\S*etcd\S*')
+----
+
+.. Take a snapshot of the etcd data in the pod and store it locally:
++
+----
+$ oc project kube-system
+$ oc exec ${ETCD_POD} -c etcd -- /bin/bash -c "ETCDCTL_API=3 etcdctl \
+    --cert /etc/etcd/peer.crt \
+    --key /etc/etcd/peer.key \
+    --cacert /etc/etcd/ca.crt \
+    --endpoints <ETCD_EP> \ <1>
+    snapshot save /backup/etcd/snapshot.db"
```

Comment:
As my understanding，the dir saving snapshot.db could not be specialed feel free, 
because the etcdctl backup will save snapshot.db in etcd pod's inner dir indeedly, if etcd pod crashed, snapshot.db file will lost also, then we could not run etcd restore without snapshot.db, so it have to keep the snapshot.db saved in mount dir which master server's local dir.

dir1 and dir2 is available dir for saving etcd snapshot.db:
vi /etc/origin/node/pods/etcd.yaml:
.....
volumeMounts:
    - mountPath: /etc/etcd/      ====> dir1
      name: master-config
      readOnly: true
    - mountPath: /var/lib/etcd/    ======>dir2
      name: master-data
    workingDir: /var/lib/etcd

@sdodson, pls correct me if there is any misunderstand, thx

Reply:
Right, if the command is running inside the container then you'll want to be careful not to write your backup to a path that's mounted into the container. Such as /var/lib/etcd

Reply:
@sdodson, I tried '/var/lib/etcd/snapshot.db', it work well for mount to master's local dir, but '/etc/etcd/snapshot.db' fail to mount to master's local dir, is this problem or work as design? thx 
@kalexand-rh, I viewed that you wrote the '/var/lib/etcd/snapshot.db' in doc, so let's waiting for scott's response, if this is work as design, pls update the doc, if it's a problem, I will file a bug for it, and the doc needn't be updated. thx

Reply:
@gpei, Scott says that the '/etc/etcd/snapshot.db' path should be fine.

Reply:
@geliu2016, Scott says that '/etc/etcd/snapshot.db' should work. I'll update it to '/var/lib/etcd/snapshot.db' since that's **actually** working, though: https://github.com/openshift/openshift-docs/pull/12454

Reply:
@kalexand-rh, ok, I think the doc is ok, and I double confirmed that '/etc/etcd/snapshot.db' doesn't work at least in 3.11, so I will file function bug to scott about this, thanks for your great work on this issue.
@sdodson , I file a bug for this issue as we discussed above, thx
https://bugzilla.redhat.com/show_bug.cgi?id=1638673

Reply:
Thanks! I'm glad that this process is correct again. I'll ping you in all the doc bugs that I think we can close now. :)